app/views/agriculturalFlatRateScheme.scala.html

```diff
@@ -37,7 +44,7 @@
         @link(appConfig.VATNotice700_46agriculturalURL, "agriculturalFlatRateScheme.linkText", isExternal = true)
     }
 
-    @helpers.form(action = AgriculturalFlatRateSchemeController.onSubmit(), 'autoComplete -> "off") {
+    @formWithCsrf(action = AgriculturalFlatRateSchemeController.onSubmit(), 'autocomplete -> "off") {
```

Comment:
Should "autocomplete" be "autoComplete" here? It is in other files

Reply:
Good spot! I deleted that line by mistake and re-added it with a bonus typo.

Reply:
Gotcha! Once that's re-pushed we're all good to merge.